entando.json

```diff
@@ -3,7 +3,11 @@
         {
             "name": "case-service",
             "stack": "spring-boot",
-            "healthCheckPath": "/api/health"
+            "healthCheckPath": "/api/health",
+						"dbms": "mysql",
+            "ingressPath": "/cm",
```

Comment:
this is not needed, better using canonical one (autogenerated)

Reply:
ingressPath deleted